tests/import_test.go

```diff
@@ -0,0 +1,30 @@
+package tests_test
+
+import (
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"kubevirt.io/containerized-data-importer/tests/framework"
+)
+
+const (
+	testSuiteName   = "Importer Test Suite"
+	namespacePrefix = "importer"
+)
+
+var _ = Describe(testSuiteName, func() {
+	f := framework.NewFramework(namespacePrefix)
+
+	Describe("Creating standard PVC should not trigger importer", func() {
+		It("PVC should remain empty after being created", func() {
+			pvc, err := f.CreatePVCFromDefinition(f.Namespace.GetName(), framework.NewPVCDefinition("no-import", "1G", nil, nil))
+			Expect(err).ToNot(HaveOccurred())
+			// Wait a while to see if CDI puts anything in the PVC.
+			time.Sleep(framework.DefaultTimeout)
+			Expect(f.VerifyPVCIsEmpty(pvc)).To(BeTrue())
+			// Not deleting PVC as it will be removed with the NS removal.
```

Comment:
This is true, however it is better practice to explicitly delete all resources created by the test. That way, if the pvc cannot be deleted you'll be able to report a meaningful error compared to the err you see if the pvc delete fails while deleting the ns,

Reply:
So here is why I decided to let the NS removal clean up stuff. When the NS cleans up stuff, its all internal in the api server. If we do it manually, we are hammering the api server with different request, so the load on it increases, and we increase the network traffic as well. At the cost of less meaningful messages if something goes wrong during the deleting of the NS.

Reply:
There is a balance between how long it takes to run a full e2e test suite vs meaningful error messages. I lean more toward the later because if a clear err msg helps QE or a developer find and fix a bug sooner I see that time saved as being very valuable. I guess this balance point is a personal choice and will likely fluctuate over the life of cdi.